Resync web-platform-tests/geolocation-API from upstream

marcoscaceres opened the pull request:
#### 32c8e0f399e0afaac3f4e52527c116306d7b4f5d
<pre>
Resync web-platform-tests/geolocation-API from upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=248294">https://bugs.webkit.org/show_bug.cgi?id=248294</a>
rdar://102634893

Syncs to 584cd49

Reviewed by Tim Nguyen.

* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/PositionOptions.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/PositionOptions.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_IDL.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_IDL.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_deny.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/idlharness.https.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/idlharness.https.window.js:
(window.onload.async idl_test):
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/non-fully-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/permission.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/permission.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/support.js:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_permission_deny.https.html:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/geolocation.idl: Renamed from LayoutTests/imported/w3c/web-platform-tests/interfaces/geolocation-API.idl.

Canonical link: <a href="https://commits.webkit.org/257006@main">https://commits.webkit.org/257006@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/96cd7931a2c9c4be19932df72c68b9445dc23daa

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/97557 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/6811 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/30724 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/107058 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/167321 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/7154 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/35567 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/89955 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/103724 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/103209 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/5354 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/84188 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/32363 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/87229 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/89021 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/75287 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/810 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/20479 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/797 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/21957 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4827 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/5613 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/44427 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/2050 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/41337 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Please add upstream revision in commit message.

marcoscaceres committed:
Resync web-platform-tests/geolocation-API from upstream
https://bugs.webkit.org/show_bug.cgi?id=248294
rdar://102634893

Syncs to 584cd49

Reviewed by Tim Nguyen.

* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/META.yml:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/PositionOptions.https-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/PositionOptions.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/clearWatch_TypeError.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/disabled-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute-redirect-on-load.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy-attribute.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/enabled-on-self-origin-by-feature-policy.https.sub.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_IDL.https-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_IDL.https.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.https.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_TypeError.html.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_allow.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/getCurrentPosition_permission_deny.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/idlharness.https.window-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/idlharness.https.window.js:
(window.onload.async idl_test):
(window.onload):
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/non-fully-active.https.html:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/permission.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/permission.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/support.js:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_TypeError.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/geolocation-API/watchPosition_permission_deny.https.html:
* LayoutTests/imported/w3c/web-platform-tests/interfaces/geolocation.idl: Renamed from LayoutTests/imported/w3c/web-platform-tests/interfaces/geolocation-API.idl.

Canonical link: https://commits.webkit.org/257006@main
